rpa-manager/modules/ROOT/pages/processautomation-prepare-project.adoc

```diff
@@ -0,0 +1,28 @@
+= Preparing Projects
```

Comment:
```suggestion
= Preparing Projects for Process Automation
```

Reply:
Process Automation is contained in parent node in navigation

Reply:
The reason I suggest adding this extra bit of info to the title in the .adoc is for SEO.